pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -209,10 +218,7 @@ void processArguments() throws Exception {
         protected String functionName;
         @Parameter(names = "--className", description = "The function's class name")
         protected String className;
-        @Parameter(
-                names = "--jar",
-                description = "Path to the jar file for the function (if the function is written in Java)",
-                listConverter = StringConverter.class)
+        @Parameter(names = "--jar", description = "Path to the jar file for the function (if the function is written in Java). It also supports url-path (http/https/file) from which worker can download the package.", listConverter = StringConverter.class)
         protected String jarFile;
         @Parameter(
                 names = "--py",
```

Comment:
The same should be applicable to python as well(as well as changes to be made in python instance to support it.